nuget/Microsoft.AspNetCore.AspNetCoreModule.props

```diff
@@ -3,8 +3,10 @@
   <PropertyGroup>
     <AspNetCoreModuleX64Location>$(MSBuildThisFileDirectory)..\contentFiles\any\any\x64\aspnetcore.dll</AspNetCoreModuleX64Location>
     <AspNetCoreModuleX86Location>$(MSBuildThisFileDirectory)..\contentFiles\any\any\x86\aspnetcore.dll</AspNetCoreModuleX86Location>
-    <RequestHandlerX64Location>$(MSBuildThisFileDirectory)..\contentFiles\any\any\x64\aspnetcorerh.dll</RequestHandlerX64Location>
-    <RequestHandlerX86Location>$(MSBuildThisFileDirectory)..\contentFiles\any\any\x86\aspnetcorerh.dll</RequestHandlerX86Location>
+    <InProcessRequestHandlerX64Location>$(MSBuildThisFileDirectory)..\contentFiles\any\any\x64\aspnetcore_inprocess_rh.dll</InProcessRequestHandlerX64Location>
+    <InProcessRequestHandlerX86Location>$(MSBuildThisFileDirectory)..\contentFiles\any\any\x86\aspnetcaspnetcore_inprocess_rhorerh.dll</RequestHandlerX86Location>
+    <OutOfProcessRequestHandlerX64Location>$(MSBuildThisFileDirectory)..\contentFiles\any\any\x64\aspnetcore_outofprocess_rh.dll</OutOfProcessRequestHandlerX64Location>
```

Comment:
I would remove _rh suffix.

Reply:
@shirhatti @pan-wang  I agree with Pavel, you fine with removing _rh?

Reply:
and make it aspnetcorev2_